src/utils.ts

```diff
@@ -55,6 +57,15 @@ export function getRootDirectory(file: File): string {
   return path.normalize(dir).slice(root.length).split(path.sep)[0];
 }
 
+/**
+ * Generates a public and private key from a provided, secure seed.
+ * @param seed - A secure seed.
+ */
+export function keyPairFromSeed(seed: string): { publicKey: PublicKey; privateKey: SecretKey } {
+  seed = pkcs5.pbkdf2(seed, "", 1000, 32, md.sha256.create());
```

Comment:
Can we add a comment on why is this needed here ? @peterjanbrone maybe you could collaborate on that